### PR TITLE
Change tor_exit_nodes feed to use the Tor Onionoo API + introduce tests

### PR DESCRIPTION
**Pull request by itsmvd**
and move test helper files to a separate module

### CODE REVIEW

File: tests/analytics_test.py [hunk 1]
```diff
@@ -13,48 +13,11 @@
 from core.schemas.indicator import DiamondModel
 from core.schemas.observable import ObservableType
 from plugins.analytics.public import censys, expire_tags, shodan
+from core.database_arango import ArangoYetiConnector
```
Comment (itsmvd) [inline tests/analytics_test.py:16]: Needs to be removed

File: tests/feeds.py [hunk 2]
```diff
@@ -20,47 +21,52 @@ def setUpClass(cls) -> None:
         database_arango.db.connect(database="yeti_test")
         database_arango.db.clear()
 
-    def test_feodo_tracker_ip_blocklist(self):
-        defaults = feodo_tracker_ip_blocklist.FeodoTrackerIPBlockList._defaults.copy()
-        defaults["name"] = "FeodoTrackerIPBlocklist"
-        feed = feodo_tracker_ip_blocklist.FeodoTrackerIPBlockList(**defaults)
-        feed.run()
+    # def test_feodo_tracker_ip_blocklist(self):
```
Comment (itsmvd) [inline tests/feeds.py:24]: Need to uncomment all these, was for debug purposes

Comment (tomchop) [reply tests/feeds.py:24]: Whenever you feel ready :)

File: tests/helpers.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional
+from core.schemas import observable
+import unittest
+from core.database_arango import ArangoYetiConnector
+
+
+class TestHelpers(unittest.TestCase):
```
Comment (tomchop) [inline tests/helpers.py:7]: Maybe rename this class to something a bit more intuitive (e.g. `YetiTestCase`?)

File: tests/tor_exit_nodes_test.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
+from unittest.mock import patch, MagicMock
+from core import database_arango
+from core.schemas.observable import ObservableType
+from tests.helpers import TestHelpers
+from core.schemas.observables import ipv4
+from plugins.feeds.public.tor_exit_nodes import TorExitNodes
+
+
+class TorExitNodesTest(TestHelpers):
```
Comment (tomchop) [inline tests/tor_exit_nodes_test.py:9]: If you follow the recommendation above, you should also change this class' base class

**Review by tomchop — CHANGES_REQUESTED** [review]
Minor changes, otherwise this looks good! Thanks :)

**Review by tomchop — APPROVED** [review]
Thanks a lot!

**Pull request merged**